### DISCUSSION TITLE
求问

### DISCUSSION THREAD

**Discussion by 10100814** [share121/inter-knot#1778] [其他]
C4途径PEP羧化酶的活性被Mal和Asp抑制是为啥(高中生竞生，偏自学，查文献没看明白来问问，不方便的话我就会把帖子删了，谢谢理解🙏)

**Comment by xiex2024** [comment]
好巧我也是生竞的，我没找到这俩抑制PEPC的原话，但我猜测应该是反馈抑制。PEPC的产物是草酰乙酸，草酰乙酸在下一步反应中根据不同的C4类型（参照武维华）会产生Mal或者Asp

**Reply by 10100814** [reply]
原话在王小菁的第八版植物生理学第93页下方C4途径调节里，就是说他这是负反馈抑制初始反应的酶是吧

**Reply by xiex2024** [reply]
是的，而且这酶的调节还有挺多方式的，可以看一下这篇论文：https://doi.org/10.3390/plants10091887

**Reply by xiex2024** [reply]
这篇文章和书上那句话直接相关：https://doi.org/10.1042/bj3070807

**Reply by 10100814** [reply]
好的好的，谢谢大佬

**Comment by hdaihdsu** [comment]
1. PEP羧化酶是C4光合作用途径中的关键酶，它催化磷酸烯醇丙酮（PEP）与二氧化碳（CO$_2$）反应生成草酰乙酸（OAA），这是C4植物固定CO2的第一步。
2. Mal（丙二酸）和Asp（天冬氨酸）是C4途径中的中间产物，它们能够反馈抑制PEP羧化酶的活性，这种抑制可能是通过与酶的调节位点结合，改变酶的构象，从而降低其活性。
3. 这种反馈抑制机制的生物学意义在于调节C4光合作用的效率，避免过量固定CO2导致的能量浪费，并在植物体内维持代谢物的平衡。
仅作参考，来自ai总结CO2为二氧化碳，因为打不出来。
